bedrock/firefox/templates/firefox/content-blocking-tour/index.html

```diff
@@ -1,152 +0,0 @@
-{% block body_class %}{% endblock %}
-
-{% block site_header %}{% endblock %}
-{% block global_nav %}{% endblock %}
-
-{% block content %}
-<main role="main">
-  <header>
-    <div class="inner-container">
-      <h1>{{ _('Content Blocking') }}</h1>
-      <p class="prefs-link">
-        {{ _('Want to turn off this feature? <a href="%s">Visit Privacy Preferences</a>')|format('https://support.mozilla.org/en-US/kb/content-blocking/') }}
-      </p>
-    </div>
-  </header>
-
-  <div class="container">
-    <div id="dummy-content">
-      <noscript>
-        <p>{{ _('Please turn on JavaScript to display this page correctly.') }}</p>
-      </noscript>
-      <div class="primary-col">
-        <div class="dubious">
-          <iframe src="https://trackertest.org/" height="100%" width="100%" frameborder="0"></iframe>
-        </div>
-        <article role="presentation"></article>
-      </div>
-      <div class="secondary-col">
-        <aside role="presentation"></aside>
-        <div class="dubious-container">
-          <div class="dubious">
-            <iframe src="https://trackertest.org/set_cookie.html" height="100%" width="100%" frameborder="0"></iframe>
-          </div>
-          <section id="info-panel" class="hidden">
-            <header tabindex="-1">
-              <h2>{{ _('Differences to expect') }}</h2>
-              {% if l10n_has_tag('remove_fastblock_1499589') %}
-              <p>{{ _('Content blocking can keep parts of pages or entire pages from loading.') }}</p>
-              {% else %}
-              <p>{{ _('Content blocking can help pages load faster, but it can also keep parts of pages or entire pages from loading.') }}</p>
-              {% endif %}
-              <button class="close-btn" type="button">{{ _('Close') }}</button>
-            </header>
-            <footer>{{ _('2 of 3') }} <button type="button">{{ _('Next') }}</button></footer>
-          </section>
-        </div>
-        <aside role="presentation"></aside>
-      </div>
-    </div>
-
-    <section id="end-state" class="hidden">
-      <img src="{{ static('img/firefox/tracking-protection/shield-tab.svg') }}" width="145" alt="">
-      <h2>{{ _('Thanks for learning about Content Blocking.') }}</h2>
-      {% if l10n_has_tag('content_blocking_65_1511420') %}
-        <p>{{ _('<a rel="external" href="%s">Learn more</a> about how it works.')|format('https://support.mozilla.org/en-US/kb/content-blocking/') }}</p>
-      {% else %}
-        <p>{{ _('Learn more about how it works by visiting the <a rel="external" href="%s">FAQ page</a>.')|format('https://support.mozilla.org/en-US/kb/content-blocking/') }}</p>
-      {% endif %}
-      <button id="reload-btn" type="button" class="button">{{ _('Restart Tour') }}</button>
-    </section>
-  </div>
-
-</main>
-{% endblock %}
-
-{% block site_footer %}{% endblock %}
-
-{% block email_form %}{% endblock %}
-
-{% block site_js %}
-  {{ js_bundle('tracking-protection-tour') }}
```

Comment:
Please remove this bundle if it's no longer used.

https://github.com/mozilla/bedrock/blob/master/media/static-bundles.json

These files can be removed too:

- https://github.com/mozilla/bedrock/blob/master/media/js/firefox/tracking-protection-tour-init.js
- https://github.com/mozilla/bedrock/blob/master/media/js/firefox/tracking-protection-tour.js
- https://github.com/mozilla/bedrock/blob/master/tests/unit/spec/firefox/tracking-protection-tour.js

And remove them from the test runner:

- https://github.com/mozilla/bedrock/blob/master/tests/unit/karma.conf.js#L39
- https://github.com/mozilla/bedrock/blob/master/tests/unit/karma.conf.js#L63